packages/partykit/src/config.ts

```
@@ -169,9 +185,36 @@ export async function fetchUserConfig(
   throw new Error("Login failed.");
 }
 
+export async function createClerkClientSession() {
+  return createClerkSession({ mode: "cli" });
+}
+
+export async function createClerkServiceTokenSession() {
+  return createClerkSession({ mode: "token" });
+}
+
+export async function fetchUserConfig(
+  method: LoginMethod
+): Promise<UserConfig> {
+  // TODO: Remove when GitHub login is deprecated
+  if (method === "github") {
+    return signInWithGitHub();
+  }
+
+  return createClerkClientSession();
+}
+
 export async function logout() {
-  if (fs.existsSync(USER_CONFIG_PATH)) {
+  const config = readUserConfig(USER_CONFIG_PATH);
+  if (config) {
     fs.rmSync(USER_CONFIG_PATH);
+    if (config.type === "clerk") {
+      try {
+        await expireClerkClientToken(config.access_token);
+      } catch (e) {
+        logger.info(chalk.dim("You were logged out this device."));
```

This is stupid... it really doesn't matter that the session is not deleted, so we don't want to show a big scary warning.

But at the same time I don't want to silently ignore this error, in case something breaks in the implementation, we should notice it.

What we really need to do is integrate an error reporting service to push these silent errors into.